src/time_parser.rs

```diff
@@ -201,22 +199,29 @@ impl TimeParser {
     }
 }
 
-pub async fn natural_parser(time: &str, timezone: &str) -> Option<i64> {
-    Command::new(&*PYTHON_LOCATION)
-        .arg("-c")
-        .arg(include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/dp.py")))
-        .arg(time)
-        .arg(timezone)
-        .arg(&*LOCAL_TIMEZONE)
-        .output()
-        .await
-        .ok()
-        .and_then(|inner| {
-            if inner.status.success() {
-                Some(from_utf8(&*inner.stdout).unwrap().parse::<i64>().unwrap())
-            } else {
-                None
-            }
-        })
-        .and_then(|inner| if inner < 0 { None } else { Some(inner) })
+pub async fn natural_parser(time: &str, _timezone: &str) -> Option<i64> {
+    println!("natural_parser of {}", time);
+    let evt = to_event(time);
+    println!("to_event: {}", evt.to_string());
```

Comment:
Worth either removing these or switching to "info!"/"debug!" before merging (I know this is still WIP, just commenting so its not forgotten :))

Reply:
For sure. I'll update them to `debug!`.